src/main/java/io/takari/aether/connector/AetherRepositoryConnector.java

```diff
@@ -489,10 +489,15 @@ public void run() {
     }
 
     private boolean resourceExist(String uri) throws IOException {
-      if (aetherClient.head(uri).getStatusCode() == 200) {
-        return true;
+      Response head = aetherClient.head(uri);
+      try {
+        if (head.getStatusCode() == 200) {
```

Comment:
what do you think to simplify it with return head.getStatusCode() == 200; directly?